sdk/identity/identity/test/public/node/clientSecretCredential.spec.ts

```diff
@@ -82,7 +82,8 @@ describe("ClientSecretCredential", function() {
     })
   );
 
-  it("supports specifying the regional authority", async function(this: Context) {
+  // This test is extremely slow. Let's skip it for now.
+  it.skip("supports specifying the regional authority", async function(this: Context) {
```

Comment:
This takes too much time to complete even on playback. Figuring out how to improve this is outside of this PR. I believe we should skip this for now.

Reply:
I’m doing a slightly better fix here: https://github.com/Azure/azure-sdk-for-js/pull/15999

Reply:
you might want to keep non-core changes in a separate PR.